iocuddle/src/lib.rs

```diff
@@ -50,7 +50,11 @@
 //! use std::os::raw::{c_void, c_int, c_uint};
 //! use iocuddle::*;
 //!
-//! let mut file = std::fs::File::open("/dev/tty").unwrap();
+//! let file = std::fs::File::open("/dev/tty");
+//! let mut file = match file {
+//!     Ok(f) => f,
+//!     Err(_) => std::process::exit(1),
+//! };
```

Comment:
```suggestion
//! let mut file = match std::fs::File::open("/dev/tty") {
//!     Err(_) => std::process::exit(1),
//!     Ok(f) => f,
//! };
```

I find that more readable.

Reply:
why not `unwrap_or_else`???

Reply:
That would indeed have been better. PR welcome. :)

Reply:
https://github.com/enarx/enarx/pull/491